app/controllers/cases_controller.rb

```diff
@@ -114,7 +115,7 @@ def undo
       end
       flash[:success] = 'Undid that!'
       flash[:undo] = @this_case.versions
-    rescue
+    rescue StandardError
```

Comment:
Let's not rescue all of `StandardError`

Reply:
OK. This can be modified to handle the specific sets of errors expected. Does that work better @rlgreen91 ?

Reply:
Yes, it does.